docs/overview/support.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Support
+last_update:
+  date: 24 January 2024
+---
+
+There are many places that you can use to reach out for developer support, including:
+
+- Tezos Discord: https://discord.gg/yXaPy6s5Nr
+- Tezos sub-reddit: https://www.reddit.com/r/tezos/
+- Tezos Stack Exchange: https://tezos.stackexchange.com
+- Tezos Element community: https://riot.tzchat.org/
+- #tezos channel on Freenode
+
+For more social media links, see https://tezos.com/community/.
+
```

Comment:
I think it would be better to start with the other docs and mention interactive support only after. This could filter the use of interactive resources, which are scarce, especially if we scale up as planned.

Reply:
Yes, good idea, thanks for the additions.

Reply:
I see that the Documentation section still comes after the Social media section. My suggestion was to swap them, so people at least see the list of all doc sites first before asking for support.